### DISCUSSION TITLE
Can't change the name of series in Charts

### DISCUSSION THREAD

**Discussion by hitfm00** [mantinedev/mantine#5583] [Feature requests]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.4.1

### What package has an issue?

@mantine/charts

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

For example:
**Data**
`[{date: "00:00", income: 2}, {date: "01:00", income: 3}, {date: "02:00", income: 95},…]`

**Component:**
```
<LineChart
    h={300}
    data={data}
    dataKey="date"
    series={[{ name: 'income', color: 'white' }]}
/>
```
And this name in the `series` prop must necessarily match the name in the `data`, I would like to suggest that you can add your own name, and not just by the key in the object

**Example**:
`series={[{ name: 'income', color: 'white', label: 'All money' }]}
`
Because for now I can't translate this name with my `i18next` library 

> P.S. It's possible to do with custom tooltip component and `tooltipProps={{ content: <CustomTooltip/> }}`

### Possible fix

Add new prop to series object
```
export interface ChartSeries {
    name: string;
    color: MantineColor;
    label?: string // Will show this if exists in tooltip
}
```

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by soham2k06** [comment]
Hello,

Is your problem about not being able to see the name you have added or you are not able to see the whole chart.

Also, please show your whole code because I tried code and data that you shared with us and it worked fully as expected.

My code:

```
import { LineChart } from "@mantine/charts";

export const data = [
  { date: "00:00", income: 2 },
  { date: "01:00", income: 3 },
  { date: "02:00", income: 95 },
];

export default function Demo() {
  return (
    <LineChart
      h={300}
      data={data}
      dataKey="date"
      series={[{ name: "income", color: "yellow" }]}
    />
  );
}
```

Thank you.

**Comment by hitfm00** [comment]
@soham2k06  Hey, everything works, I just suggest that we can change the name in the tooltip, at the moment we can only show what comes from the request, and this is `income` (as shown in the screenshot), but instead of income I want to see a different name, which is what I I'll write `t('translatedText')` from `i18next`, for example

<img width="309" alt="Снимок экрана 2024-01-13 в 17 31 24" src="https://github.com/mantinedev/mantine/assets/21322208/cdca3eca-2aed-4fd1-a8da-31f7a6e112b6">

At the moment, in the name of tooltip we only see the key in the object, and not what I need.
The keys may not have the name needed to be displayed to the user

I hope I expressed myself correctly :) Thanks

<img width="310" alt="Снимок экрана 2024-01-13 в 17 37 02" src="https://github.com/mantinedev/mantine/assets/21322208/39fbc514-b57f-4519-ab52-76d03685a414">

Like on this screen

**Reply by bastiengrignon** [reply]
@hitfm00 This is actually possible with the latest version of the [LineChart](https://mantine.dev/charts/line-chart/#series-labels)
